frontend/package.json

```diff
@@ -10,6 +10,7 @@
     "react-bootstrap": "^0.32.4",
     "react-dom": "^16.7.0",
     "react-localization": "^1.0.13",
+    "react-medium-image-zoom": "^3.0.15",
```

Comment:
This does cause a warning when build dependencies for the first time:
` warning " > react-medium-image-zoom@3.0.15" has unmet peer dependency "prop-types@^15.5.8".`

Usually fixing this is as easy as adding `"prop-types":"^15.5.8",` on the next line

Reply:
Basically this is a "dependency" but it needs to be defined by our project. I think this is also to prevent crazy dependency collisions as peer dependencies see to be common for many projects, and making them peer prevents all the projects from explicitly defining the version and causing conflicts? (at least this is roughly my understanding)

Reply:
Just as a note to document it here: we did check that prop-types@15.5.8 was clean of known vulnerabilities at this time: https://snyk.io/test/npm/@types/prop-types/15.5.8

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/137/commits/e6f8f410b7c528397a1908327407a22a87a9d0b1